maintainers-guide.md

```diff
@@ -195,34 +195,40 @@ issues.
 *   [`team-Android`](https://github.com/bazelbuild/bazel/labels/team-Android): Issues for Android team
     *   Contact: [ahumesky](https://github.com/ahumesky)
 *   [`team-Apple`](https://github.com/bazelbuild/bazel/labels/team-Apple): Issues for Apple team (also MacOS, iOS etc)
-    * Contact: [aragos](https://github.com/aragos)
+    * Contact: ?
+*   [`team-Bazel`](https://github.com/bazelbuild/bazel/labels/team-Bazel): General Bazel product/strategy issues
+    * Contact: ?
```

Comment:
Suggest joehicks@google.com

Reply:
+1

Reply:
@hicksjoseph

Reply:
email vs github handle gets into the question of why we have owners here. 
We don't really need team owners for users to assign issues, since we don't expect users to know what team owns the problem.
To me, the team owners are an escalation contact. If you have an issue that got assigned to a team, and you think it has gone stale, you can directly reach them. For that reason, I favor real email addresses, where the contact is not intermediated by github.

Reply:
Since you can put your email address on your GitHub profile, I think the GitHub handle is superior. It's convenient for publicly cc'ing or assigning to someone.

Reply:
We need at least one GitHub handle for each team for assigning PRs.

Reply:
I may have missed something in the discussion today, but I'm a little confused about the issue of github events being missed. My settings (which are presumably the defaults) notify me if I'm @mentioned or subscribed in some way to the issue/PR. Could we just request that the whole team make sure that's true for them? Then the github handle/email question goes away, I think.

Reply:
Janak, the current triage policy is to label bugs with team-XXX and **not** assign it to a specific individual.
I personally would like to get notified for every bug related to team-Performance but I don't know if there's consensus among the TLs.